Dockerfiles/midstream/README.md

```diff
@@ -15,18 +15,18 @@ First thing you reckon is that you are running this command directly from upstre
 
 ```
 git clone https://github.com/redhat-operator-ecosystem/operator-test-playbooks.git
-cd Dockerfiles
-### podman build -t <image_name>:<tagname> -f <filepath> --build-arg OPERATOR_SDK_VERSION=<operator_sdk_version> --build-arg OPERATOR_TEST_PLAYBOOKS_TAG=<operator_test_playbooks_tag>
+cd operator-test-playbooks
+### podman build -t <image_name>:<tagname> -f <filepath> --build-arg OPERATOR_SDK_VERSION=<operator_sdk_version> --build-arg
 ### example:
-podman build -t midstream_image:latest -f Dockerfile --build-arg OPERATOR_SDK_VERSION=v1.4.0 --build-arg OPERATOR_TEST_PLAYBOOKS_TAG=v1.0.11
+podman build -t midstream_image:latest -f Dockerfile . --build-arg OPERATOR_SDK_VERSION=v1.4.0 
 ```
 
 ## Running image with operator bundle
 
 ```
-podman run -it -v <operator_bundle_dir>:/project/operator-bundle -v <output_log>:/project/output --security-opt label=disable <imagename>:<tagname> -e IMAGE_TO_TEST=<image_url_to_test>
+podman run -it -v <operator_bundle_dir>:/project/operator-bundle -v <output_log>:/project/output --security-opt label=disable <imagename>:<tagname> -e IMAGE_TO_TEST=<image_url_to_test> -e VERBOSITY=<number in range of 1 to 4>
```

Comment:
Why do we need to bind-mount `<operator_bundle_dir>:/project/operator-bundle` ?

Reply:
removed. We do not need it.